src/consul/tests/test.bal

```diff
@@ -29,17 +29,18 @@ ConsulConfiguration consulConfig = {
 
 Client consulClient = new(consulConfig);
 
-@test:Config
+@test:Config {
+}
 function testRegisterService() {
     io:println("--------------Calling registerService----------------");
     json jsonPayload = {"ID":"redis", "Name":"redis1", "Address":"localhost", "port":8000, "EnableTagOverride":false};
-    var serviceRegister = consulClient->registerService(jsonPayload);
+    boolean|error serviceRegister = consulClient->registerService(jsonPayload);
 
     if (serviceRegister is boolean) {
         test:assertEquals(serviceRegister, true, msg = "Failed to call registerService()");
     } else {
-        io:println(<string>serviceRegister.detail().message);
-        test:assertFail(msg = <string>serviceRegister.detail().message);
+        string? val = serviceRegister.detail()?.message;
+        test:assertFail(msg = (val is string)? val: "failed to register service call!" );
```

Comment:
L42 and L43 can be combined with the elvis operator
```suggestion
        test:assertFail(msg = serviceRegister.detail()?.message ?: "failed to register service call!" );
```

Reply:
Done